client/actions/planning.js

```diff
@@ -4,18 +4,26 @@ import * as actions from '../actions'
 import { pickBy, cloneDeep, isNil, has, get } from 'lodash'
 
 const createAgenda = ({ name }) => (
-    (dispatch, getState, { api, notify }) => {
+    (dispatch, getState, { api, notify }) =>
         api('planning').save({}, {
             planning_type: 'agenda',
             name: name,
         })
         .then((agenda) => {
-            notify.success('An agenda has been added')
+            notify.success('An Agenda has been added.')
             dispatch(hideModal())
             dispatch(addOrReplaceAgenda(agenda))
             dispatch(selectAgenda(agenda._id))
+        }, (error) => {
```

Comment:
We used to raise a `SubmissionError` in order to show the error in the form.
It's part of redux-form, you can have a look there: http://redux-form.com/6.6.3/examples/submitValidation/